src/app/register2/components/form-personal-additional-emails/form-personal-additional-emails.component.html

```diff
@@ -29,12 +29,12 @@
     <mat-error
       *ngIf="
         (additionalEmailsTouched &&
```

Comment:
MatError will be show only if the control was touched AND it has an error. 

With the parentheses changes you are showing the error immediately if the pattern is not correct, right? I believe we want to wait until the user focuses out the control to show the error.